package.json

```diff
@@ -63,41 +62,42 @@
     "core-js": "3.22.8",
     "cross-spawn": "^7.0.3",
     "css-loader": "^6.7.3",
-    "css-minimizer-webpack-plugin": "^3.4.1",
+    "css-minimizer-webpack-plugin": "^5.0.0",
     "dotenv": "^16.0.3",
-    "dotenv-expand": "^8.0.3",
-    "eslint": "^8.35.0",
+    "dotenv-expand": "^10.0.0",
+    "eslint": "^8.39.0",
     "eslint-config-enact": "^4.1.4",
     "eslint-plugin-enact": "^1.0.2",
-    "eslint-plugin-jest": "^26.6.0",
+    "eslint-plugin-jest": "^27.2.1",
```

Comment:
Is this okay not to match the version from `eslint-config-enact`'s?

Reply:
We will check this in WRP-14712